src/main/kotlin/com/shortener/domain/UrlEntryRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.shortener.domain
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+
+@Repository
+interface UrlEntryRepository : JpaRepository<UrlEntry, Long> {
+    fun findAllByOrderByIdDesc(): List<UrlEntry>
```

Comment:
Name of `findAllByOrderBy`: what means `findAllBy`? By what?

Is that a typo, because i don't understand what the function is supposed to do.

Reply:
Is not a typo. This is how the jpa is deriving the query from the method name. By not specifying the attribute(`findAllBy[attribute]..`, I am expecting no `where` clause to be added, therefore, all entries will be fetched, reversed ordered by id. Note: `findAllOrderByIdDesc` does not work, already tried

Reply:
You can jus use a Sort parameter instead of hard coding the value in this function

Reply:
And why should I then fill provide this Sort parameter when I don't need it ?